pkgs/development/python-modules/keyring/default.nix

```diff
@@ -34,7 +34,23 @@ buildPythonPackage rec {
     secretstorage
   ];
 
-  pythonImportsCheck = [ "keyring" "keyring.backend" ];
+  pythonImportsCheck = [
+    "keyring"
+    "keyring.backend"
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  # Keychain communications isn't possible in our build environment
+  # keyring.errors.KeyringError: Can't get password from keychain: (-25307, 'Unknown Error')
+  disabledTests = lib.optionals (stdenv.isDarwin) [
+    "test_multiprocess_get"
+    "test_multiprocess_get_after_native_get"
+  ];
+
+  disabledTestsPaths = lib.optional (stdenv.isDarwin) "tests/backends/test_macOS.py";
```

Comment:
This line doesn't make much sense. I doubt they are executed on Linux, so why not always disable them?

Reply:
Agree